.github/workflows/ci.yml

```diff
@@ -381,6 +382,10 @@ jobs:
         yarn-project-formatting,
         yarn-project-test,
         prover-client-test,
+        bb-js-test,
+        barretenberg-acir-tests-bb-js,
+        barretenberg-acir-tests-bb,
+        barretenberg-acir-tests-sol,
```

Comment:
I'm thinking whether you could just have `notify` depend on `merge-check` so we don't have to duplicate this list